src/log/log.c

```diff
@@ -1365,10 +1369,10 @@ __wti_log_acquire(WT_SESSION_IMPL *session, uint64_t recsize, WTI_LOGSLOT *slot)
      * risk of an error due to no space.
      */
     if (F_ISSET(log, WTI_LOG_FORCE_NEWFILE) || !__log_size_fit(session, &log->alloc_lsn, recsize)) {
-        WT_RET(__log_newfile(session, false, &created_log));
-        F_CLR(log, WTI_LOG_FORCE_NEWFILE);
-        if (log->log_close_fh != NULL)
+        WT_RET(__log_newfile(session, false, &created_log, &log_to_close));
```

Comment:
Why did the structure of this need to change? I've spent barely any time in the logging code so apologies if I've missed something obvious.

Reply:
Thanks, that helped me understand this part of the change.